geometry_msgs/Point[] left_1
geometry_msgs/Point[] left_0
geometry_msgs/Point[] right_0
geometry_msgs/Point[] right_1
float32[] coeff_left_1
float32[] coeff_left_0
float32[] coeff_right_0
float32[] coeff_right_1
